python/hail/dataset.py

```diff
@@ -3408,7 +3408,7 @@ def min_rep(self, left_aligned=False):
     @typecheck_method(k=integral,
                       compute_loadings=bool,
                       as_array=bool)
-    def normalized_genotype_pca(self, k=10, compute_loadings=False, as_array=False):
+    def pca_normalized_genotypes(self, k=10, compute_loadings=False, as_array=False):
```

Comment:
This seems unnecessary.  I also prefer the former.  We don't say "dog big", we say "big dog".

Reply:
I'm thinking of PCA as a verb like "dimensionally reduce", as it's sometimes used. How about `pca_on_normalized_genotypes` or `pca_with_normalized_genotypes`. All things being equal, it's also nicer for this command to be consecutive with `pca` in docs.

If you feel strongly, I'll change back to `normalized_genotype_pca` but I find that hard to parse.